cfApplication.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+type Application struct {
+	Name      string            `json:"name"`
+	Metadata  Metadata          `json:"metadata,omitempty"`
+	Env       map[string]string `json:"env,omitempty"`
+	Services  Services          `json:"services,omitempty"`
+	Processes Processes         `json:"processes,omitempty"`
+	Replicas  uint              `json:"replicas,omitempty"`
+}
+
+type Metadata struct {
+	Labels      map[string]string `json:"labels,omitempty"`
+	Annotations map[string]string `json:"annotations,omitempty"`
+}
+
+type Routes []Route
+
+type Route struct {
+	URL      string        `json:"fqdn,omitempty"`
+	Protocol RouteProtocol `json:"protocol,omitempty"`
+}
+
+type RouteSpec struct {
+	Type   RouteType `json:"type"`
+	Routes Routes    `json:"routes"`
+}
+
+type RouteType string
+
+const (
+	Default RouteType = "default"
+	Random  RouteType = "random"
+)
+
+type RouteProtocol string
+
+const (
+	HTTP  RouteProtocol = "http"
+	HTTPS RouteProtocol = "https"
+	TCP   RouteProtocol = "tcp"
+)
+
+// Services represents a slice of Service
+type Services []Service
+
+// Service contains the specification for an existing Cloud Foundry service required by the application
+type Service struct {
+	// Name represents the name of the Cloud Foundry service required by the application
+	Name string `json:"name"`
+	// Parameters contain the k/v relationship for the aplication to connect to the service
+	Parameters map[string]interface{} `json:"parameters,omitempty"`
+}
+
+// Processes represents a slice of Processes
+type Processes []Process
```

Comment:
So do we keep `Processes` name?

Reply:
Just for consistency purposes. We are capturing the CF application values, I didn't think at this point worth converting it to K8s naming convention because that should be done at transformation.

Reply:
I'm going to align the health and readiness probe names to CF convention. I should be consistent with my own statements 😅

Reply:
Updated